### ISSUE TITLE
weight of CrossEntropyLoss

### ISSUE THREAD

**Issue by nationalflag** [pytorch/pytorch#4192]
When I set weight=torch.Tensor([1, 0.3]), sometimes loss decreass but sometimes loss increase compared to weight=torch.Tensor([1, 1]).

**Comment by apaszke** [comment]
This might happen. That's not enough to show that there's a bug in our code. We're using GitHub issues for bug reports only, and all questions should be posted on [our forums](discuss.pytorch.org).
